drivers/storage/portworx/csi.go

```diff
@@ -23,17 +30,38 @@ type csiVersions struct {
 	// Before Kube v1.13 the registration directory must be /var/lib/kubelet/plugins instead
 	// of /var/lib/kubelet/plugins_registry
 	useOlderPluginsDirAsRegistration bool
+	// driverName is decided based on the PX Version unless the deprecated flag is provided.
+	driverName string
+	// useDeployment decides whether to use a StatefulSet or Deployment for the CSI side cars.
+	useDeployment bool
+	// includeAttacher dictates whether we include the csi-attacher sidecar or not.
+	includeAttacher bool
+	// includeResizer dicates whether or not to include the resizer sidecar.
+	includeResizer bool
+	// includeCsiDriverInfo dictates whether or not to add the CsiDriverInfo object.
+	includeCsiDriverInfo bool
```

Comment:
It doesn't look like this is used.

If this is true, we must created a CSIDriver object. This happens in k8s 1.14+